<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>excluir_cliente_op4</class>
 <widget class="QWidget" name="excluir_cliente_op4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>459</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Excluir cliente</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>excluir_cliente_op4.png</normaloff>excluir_cliente_op4.png</iconset>
  </property>
  <widget class="QLabel" name="label_titulo_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tem certeza que deseja excluir?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sim">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Sim</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>excluir_cliente_op4.png</normaloff>excluir_cliente_op4.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_nao">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Não</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>excluir_cliente_op4_btn_nao.png</normaloff>excluir_cliente_op4_btn_nao.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_report_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_titulo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Digite o nome do cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_report">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_nome_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_adress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereço:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_valor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor em compras:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_adress">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_valor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_buscar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pesquisar_cliente_op3.png</normaloff>pesquisar_cliente_op3.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
